utils/utils.go

```diff
@@ -41,4 +42,36 @@ func GetAbsPath(path string) string{
 		}
 	}
 	return result
-}
\ No newline at end of file
+}
+
+func GetCurrentServerVersion() string {
+	var version string
+	//var uversion string
+	path := "/usr/share/rhn/config-defaults/rhn_web.conf"
+
+	f, err := os.Open(path)
+	if err != nil {
+		log.Fatal().Msg("Couldn't find rhn_web.conf")
+	}
+	defer f.Close()
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		if strings.Contains(scanner.Text(), "web.version = ") {
```

Comment:
Before getting the version we need to check which product we are exporting from, using the flag `product_name`